stress/common/src/main/java/alluxio/stress/worker/UfsIOParameters.java

```diff
@@ -23,6 +23,10 @@
  * Parameters used in the UFS I/O throughput test.
  * */
 public class UfsIOParameters extends Parameters {
+  public static final String USE_UFS_CONF = "--use-ufs-conf";
```

Comment:
Can you update the names in this file to use these constants?

Reply:
done

Reply:
NIT: I think something like `--is-mounted` or `--use-mount-conf` makes a bit more sense, since this essentially controls if the path is mounted or not.